lib/manager/npm/extract/locked-versions.spec.ts

```diff
@@ -15,7 +15,7 @@ describe('manager/npm/extract/locked-versions', () => {
       async (yarnVersion) => {
         yarn.getYarnLock.mockReturnValue({
           isYarn1: yarnVersion === '1.22.0',
-          cacheVersion: yarnVersion === '2.2.0' ? 6 : NaN,
+          lockfileVersion: yarnVersion === '2.2.0' ? 6 : undefined,
```

Comment:
Why `undefined` instead of `NaN` ?

Reply:
@viceice I wasn't sure `undefined` is converted to `NaN` in integer comparisons and worried about TypeScript errors in #7204. Now I think it seems that implicit `undefined` is cleaner than explicit `NaN`.

Reply:
ok